Code/Source/RobotImporter/URDF/PrefabMakerUtils.cpp

```diff
@@ -95,4 +97,27 @@ namespace ROS2
         AzToolsFramework::EditorEntityContextRequestBus::Broadcast(
             &AzToolsFramework::EditorEntityContextRequests::AddRequiredComponents, *entity);
     }
+
+    bool PrefabMakerUtils::HasCollider(AZ::EntityId entityId)
+    {
+        AZ::Entity* entity = AzToolsFramework::GetEntityById(entityId);
+        return entity->FindComponent<PhysX::EditorColliderComponent>() != nullptr ||
+            entity->FindComponent<PhysX::EditorShapeColliderComponent>() != nullptr;
```

Comment:
Nitpick: Technically, there is also a third type of editor collider component, `PhysX::EditorHeightfieldColliderComponent`.

Reply:
Yeah, but in fact after discussion with the physics team we are actually going to require specifically ECC (nothing else works for us)